exchanges/credentials_test.go

```diff
@@ -55,8 +56,8 @@ func TestGetCredentials(t *testing.T) {
 
 	ctx = context.WithValue(context.Background(), account.ContextCredentialsFlag, "pewpew")
 	_, err = b.GetCredentials(ctx)
-	if !errors.Is(err, errContextCredentialsFailure) {
-		t.Fatalf("received: %v but expected: %v", err, errContextCredentialsFailure)
+	if !errors.Is(err, common.ErrTypeAssertFailure) {
+		t.Fatalf("received: %v but expected error to wrap: %v", err, common.ErrTypeAssertFailure)
```

Comment:
If you ever see this before merge:
```suggestion
        require.ErrorIs(t, err, common.ErrTypeAssertFailure)
```